src/Umbraco.Core/Services/Implement/ContentService.cs

```diff
@@ -188,11 +188,34 @@ public IContent Create(string name, int parentId, string contentTypeAlias, int u
             // TODO: what about culture?
 
             var contentType = GetContentType(contentTypeAlias);
-            if (contentType == null)
-                throw new ArgumentException("No content type with that alias.", nameof(contentTypeAlias));
+            return Create(name, parentId, contentType, userId);
+        }
+
+        /// <summary>
+        /// Creates an <see cref="IContent"/> object of a specified content type.
+        /// </summary>
+        /// <remarks>This method simply returns a new, non-persisted, IContent without any identity. It
+        /// is intended as a shortcut to creating new content objects that does not invoke a save
+        /// operation against the database.
+        /// </remarks>
+        /// <param name="name">The name of the content object.</param>
+        /// <param name="parentId">The identifier of the parent, or -1.</param>
+        /// <param name="contentType">The content type of the content</param>
+        /// <param name="userId">The optional id of the user creating the content.</param>
+        /// <returns>The content object.</returns>
+        public IContent Create(string name, int parentId, IContentType contentType,
```

Comment:
I wonder if this would be better to supply the IContent as the parent (it could be null) instead of relying on this method to do the lookup. I understand that it 'could' be cached but that is not always the case. This method could be used for better performance to (for example) create a bunch of content under a single parent and not have to rely on the service to keep looking it up. Cache isn't permanent so we can't always rely on it.

Reply:
Ahh that's a very good point, however, `GetEmpty` and `GetEmptyByKey` only creates a single content, so they just pass in the parent ID, so I've added an additional overload, so we now have: 

* Create content with  parent ID and content type alias
* Create content with parent ID and content type
* Create content with parent and content type

So we cover all cases 😄

Reply:
Hmm I'm sorry but I just had a closer look at this after the tests were failing, and I don't think we can do this in the end, it turns out that the constructor differs for the `Content` depending on whether you use an ID or and `IContent`, if we use `IContent` i must not be null. 

This becomes problematic when `Create` is called with an ID of a parent that does not exists, since `IContent parent` will be null. I did consider if we could just assume that if the parent was null that the parent ID would be -1 since that's root, and call the constructor with -1 as the parent ID, however when a parent is null the ID is not always -1, for instance with the blocklist we send the ID as -20. Since the `IContent parent` would be null we have no way of knowing this and would change the ID to -1, potentially messing it up for some if they expect the ID to be -20 for some reason.